tests/e2e/e2e.go

```diff
@@ -275,10 +275,10 @@ func GetBasicAuth(ctx context.Context, name string) (BasicAuth, error) {
 				continue
 			}
 
-			for i := range secs {
-				if secs[i].Name == name {
-					out.Password = secs[i].Data["password"]
-					out.Username = secs[i].Data["username"]
+			for _, sec := range secs {
+				if sec.Name == name {
+					out.Password = sec.Password
+					out.Username = sec.Username
 					break
```

Comment:
We are probably unmarshaing to wrong type.

Reply:
I think it's the right type, the sec slice is made from TemplateData struct.

Reply:
Changed it, Now using Secret struct from entity package